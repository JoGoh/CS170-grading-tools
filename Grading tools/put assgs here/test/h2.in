 1 0 -1 2 -3 1 1 1 -4 0 0 0 0 0 0 -3 -21