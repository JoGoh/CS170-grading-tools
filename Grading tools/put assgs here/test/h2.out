-21 -3 0 0 0 0 0 0 -4 1 1 1 -3 2 -1 0 1 

Reversing: 
1 0 -1 2 -3 1 1 1 -4 0 0 0 0 0 0 -3 -21 

Sorting: 
-21 -4 -3 -3 -1 0 0 0 0 0 0 0 1 1 1 1 2 

Uniquing: 
-21 -4 -3 -1 0 1 2 

Clearing: 

